index.html

```diff
@@ -10656,19 +10660,19 @@ <h3>User Prompt Handler</h3>
  on <var>handler</var>&apos;s [=prompt handler configuration/handler=]:
 
 <dl class=switch>
-<dt>"<code>accept</code>"
-<dd><p><a>Accept</a> the <a>current user prompt</a>.
+ <dt>"<code>accept</code>"
+ <dd><p><a>Accept</a> the <a>current user prompt</a>.
 
-<dt>"<code>dismiss</code>"
-<dd><p><a>Dismiss</a> the <a>current user prompt</a>.
+ <dt>"<code>dismiss</code>"
+ <dd><p><a>Dismiss</a> the <a>current user prompt</a>.
 
-<dt>"<code>ignore</code>"
-<dd><p>Do nothing.
+ <dt>"<code>ignore</code>"
+ <dd><p>Do nothing.
 </dl>
 
-<li><p>If <var>handler</var>&apos;s [=prompt handler
-   configuration/notify=] is true, return <a>annotated unexpected
-   alert open error</a>.
+<li><p>If <var>handler</var>&apos;s [=prompt handler configuration/notify=]
```

Comment:
Returning unexpected alert if `notify` isn't true seems like it misses the point of `notify`.

Reply:
The `ignore` handler causes all prompts to not be closed, but an error reported. Should a new case be allowed for not raising an error? This would require quite a lot of changes especially if we want to keep the backward compatibility with classic. Not sure if that is really doable.

Reply:
In that case set /notify/ to true when we get "ignore" as the capability value, rather than adding a special case here.

Reply:
Oh right. That is indeed a better approach.